### PR TITLE
Weather#11

### PR DESCRIPTION
**Pull request by venkykatta**
Code formatters.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 19.10b0 # Replace by any tag/version: https://github.com/psf/black/tags
+    hooks:
+      - id: black
+        language_version: python3 # Should be a command that runs python3.6+
```
Comment (su66uu) [inline .pre-commit-config.yaml]: This config is related to the commit hooks. That means the git will run `black` when you commit something. But what you need is that when you save anything in vs code, the vs code should run `black` and format the code. 

And I also observed that this `pre-commit-config.yaml` file only works when you install `pre-commit` package. But you have not mentioned it anywhere. You should have added that package to `requirements.txt` and commit it.
So the rule of thumb is always to let your team know what are all things you did for that particular story. In here, I don't know how `.pre-commit-config.yaml` works?

**Review by su66uu — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (venkykatta) [reply .pre-commit-config.yaml]: Yes, Subbu. You are correct. The `pre-commit-config.yaml` file only works when I install a `pre-commit` package. That was bad. I heard about the `pyproject.toml` file and I check that file, that is the **black** formatted configuration file. I add the `.toml` file to my project.

Comment (su66uu) [reply .pre-commit-config.yaml]: Great 👏 

> And I also observed that this pre-commit-config.yaml file only works when you install pre-commit package. But you have not mentioned it anywhere. You should have added that package to requirements.txt and commit it.

How are you handling this?

Comment (venkykatta) [reply .pre-commit-config.yaml]: Whenever a person is using my project that person was confused to use this `.pre-commit-config.yaml` file. So, that's why _config_. ` .toml` file to my project.

Comment (venkykatta) [reply .pre-commit-config.yaml]: And also have another way to use this `.pre-commit-config` file. That is add the `pre-commit` package to the `requirements.txt `file and commit it.

**Comment by su66uu** [comment]
As we don't find a proper way to set up Black and it is taking a lot of time, we are thinking of parking aside this story for some time

**Pull request closed without merging**